library/percona

```diff
@@ -7,8 +7,8 @@ GitRepo: https://github.com/percona/percona-docker.git
 GitFetch: refs/heads/master
 Architectures: amd64
 
-Tags: 8.0.20-11-centos, 8.0-centos, 8-centos, 8.0.20-11, 8.0, 8, ps-8.0.20-11, ps-8.0, ps-8
-GitCommit: 65855912b9946093cee42f987336440972405391
+Tags: 8.0.21-12-centos, 8.0-centos, 8-centos, 8.0.21-12, 8.0, 8, ps-8.0.21-12, ps-8.0, ps-8
+GitCommit: 387a5ec03894c49e89cb992611fa365dcaf40ff1
 Directory: percona-server.80
```

Comment:
```suggestion
Directory: percona-server-8.0
```

Reply:
Gah, I thought we had a switch to new scheme for 8.0..
Thanks, added this